scripts/cargo-install-all.sh

```diff
@@ -52,6 +52,7 @@ BINS=(
   solana-install-init
   solana-keygen
   solana-ledger-tool
+  solana-network-tool
```

Comment:
maybe call this `iftop-post-processer`?  That's what it is right?

Reply:
I was hoping to roll in more functionality into the tool. The arguments can decide what to do.

Reply:
renamed to `log-analyzer`